backend/src/main/java/com/javahelp/backend/query/VanillaConstraint.java

```diff
@@ -0,0 +1,10 @@
+package com.javahelp.backend.query;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class VanillaConstraint extends Constraint{
```

Comment:
Perhaps a better naming scheme would be `ConstraintBase` for the abstract class, and `Constraint` for this class